front/src/modules/ui/data-table/table-cell/components/TableCellContainer.tsx

```diff
@@ -58,7 +59,12 @@ export const TableCellContainer = ({
 }: EditableCellProps) => {
   const { isCurrentTableCellInEditMode } = useCurrentTableCellEditMode();
 
-  const [isHovered, setIsHovered] = useState(false);
+  const isHovered = useRef<boolean>(false);
```

Comment:
We try to avoid using useRef as state management as stated in the documentation : https://docs.twenty.com/developer/frontend/best-practices#do-not-use-useref-to-store-state

You can use a simple useState here, it works just as well.

Reply:
Okay I removed useRef so you can check again @lucasbordeau